### PR TITLE
tesseract: add recipe to build languages support packages

### PR DESCRIPTION
**Pull request by phoudoin**

### CODE REVIEW

File: app-text/tesseract_data/tesseract_data-3.04.00.recipe [hunk 1]
```diff
@@ -0,0 +1,158 @@
+SUMMARY="Languages data files for Tesseract OCR engine"
+DESCRIPTION="Tesseract OCR can be fully trained to recognize new language and script. \
+A set of files for already, community made, trained languages are available, splitted \
+in seperate package per language."
+HOMEPAGE="http://code.google.com/p/tesseract-ocr"
```
Comment (humdingerb) [inline app-text/tesseract_data/tesseract_data-3.04.00.recipe]: Some typos:
"Tesseract OCR can be fully trained to recognize new _languages_ and _scripts_. \
 A set of files for [_already, --delete_]community made, trained languages are available as \
 _separate packages per language._"

File: app-text/tesseract_data/tesseract_data-3.04.00.recipe [hunk 2]
```diff
@@ -0,0 +1,157 @@
+SUMMARY="Languages data files for Tesseract OCR engine"
```
Comment (humdingerb) [inline app-text/tesseract_data/tesseract_data-3.04.00.recipe]: "_Language_ data files for Tesseract OCR engine"

File: app-text/tesseract_data/tesseract_data-3.04.00.recipe (hunk 1 above)
Comment (phoudoin) [inline app-text/tesseract_data/tesseract_data-3.04.00.recipe]: tx for pr my poor english writing

**Comment by phoudoin** [comment]
PR updated to fix requested changes

**Comment by phoudoin** [comment]
The updated PR fix your remarks. 
Could someone review and apply it ?

**Pull request merged**